test/Exception/BadMethodCallExceptionTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\Expressive;
+
+use Zend\Expressive\Exception\BadMethodCallException;
+use Zend\Expressive\Exception\ExceptionInterface;
+
+final class BadMethodCallExceptionTest extends \PHPUnit_Framework_TestCase
+{
+    public function testException()
+    {
+        $exception = new BadMethodCallException();
+
+        $this->assertInstanceOf(ExceptionInterface::class, $exception);
+
+        $this->setExpectedException(BadMethodCallException::class);
+
+        throw $exception;
```

Comment:
As discussed before, this is not the right way to test an exception in my opinion, but I'll have to think about it.